charts/rancher-monitoring/100.1.1+up19.0.3/templates/rancher-monitoring/dashboards/rancher/performance-dashboards.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Values.grafana.defaultDashboards.namespace }}
+  name: rancher-default-dashboards-performance-debugging
+  annotations:
+{{ toYaml .Values.grafana.sidecar.dashboards.annotations | indent 4 }}
+  labels:
+    {{- if $.Values.grafana.sidecar.dashboards.label }}
+    {{ $.Values.grafana.sidecar.dashboards.label }}: "1"
+    {{- end }}
+    app: {{ template "kube-prometheus-stack.name" $ }}-grafana
+{{ include "kube-prometheus-stack.labels" $ | indent 4 }}
+data:
+{{ (.Files.Glob "files/rancher/performance/*").AsConfig | indent 2 }}
+{{- end }}
```

Comment:
This means we install the dashboard even if the performance system is not turned on. Which I suppose makes sense, because you'd turn it on/off with the env var only.

Reply:
@cbron yeah, it's not a super big deal if the data isn't there. The graphs are just empty in that case.